ClosedXML/Excel/CalcEngine/Functions/MathTrig.cs

```diff
@@ -699,6 +744,19 @@ private static object RoundUp(List<Expression> p)
             return Math.Floor(value * Math.Pow(10, digits)) / Math.Pow(10, digits);
         }
 
+        private static object Sec(List<Expression> p)
+        {
+            if (double.TryParse(p[0], out double number))
```

Comment:
Use C#6 syntax

Reply:
Thanks. I'm quite used to C#7 already, didn't even think about using it that way.